Header.html

```diff
@@ -0,0 +1,56 @@
+<html>
+    <div class="sticky">
+        <nav class="navbar navbar-inverse">
+            <div class="container-fluid">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>                        
+                </button>
+                <a href="HomePage.html"><img  src="pictures/DevilSecLogo.png"  height="50" width="50"/></a>
+            </div>
+            <div class="collapse navbar-collapse" id="myNavbar">
+                <ul class="nav navbar-nav">
+                    <li class="active"><a href="HomePage.html">Home</a></li>
+                    <li><a href="GetConnected.html">Get Connected</a></li>
+                    <li><a href="ClubRecordings.html">Club Recordings</a></li>
+                    <li><a href="Resources.html">Resources</a></li>
+                    <li><a href="DSSchedule.html">Schedule/Calendar</a></li>
+                </ul>
+            </div>
+            </div>
+        </nav>
+    </div>
+
+    <style>
```

Comment:
The appearance of the header is broken, even the navbar toggle button does not work but it does appear to rely on bootstrap classes so if that is indeed the case please say so.
Otherwise, please fix.

Reply:
It does indeed rely on bootstrap classes.